Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -50,8 +63,11 @@ def getCondorLogFile( pilotRef ):
   """return the location of the logFile belonging to the pilot reference"""
   _jobUrl, condorID = condorIDFromJobRef( pilotRef )
   #FIXME: This gets called from the WMSAdministrator, so we don't have the same
-  #working directory as for the SiteDirector unless we force it
-  resLog = findFile( DEFAULT_WORKINGDIRECTORY, '%s.log' % condorID )
+  #working directory as for the SiteDirector unless we force it, there is also
+  #no CE instantiated when this function is called so we can only pick this option up from one place
+  workingDirectory = gConfig.getValue( "Resources/Computing/CEDefaults/HTCondorCE/WorkingDirectory",
```

Comment:
Agreed with the location of the option, I didn't read the locations you added carefully enough.

This can only be set in one location, because if the default value were to be overwritten we could not find the log files when we call this function from the WMSAdministrator.